components/StyledReturn/StyledReturn.js

```diff
@@ -0,0 +1,19 @@
+import Link from "next/link";
+import Image from "next/image";
+
+export default function StyledReturn() {
+  return (
+    <>
+      <Link href="/">
+        <Image
+          src={"/price.png"}
+          width="35"
+          height="35"
+          alt="return bottom"
+          style={{ position: "fixed", top: "150px", right: "5px" }}
+          aria-label="Return to home page"
+        ></Image>
+      </Link>
+    </>
```

Comment:
you don't need fragments here and the Image can be self-closing:

```suggestion
      <Link href="/">
        <Image
          src={"/price.png"}
          width="35"
          height="35"
          alt="return bottom"
          style={{ position: "fixed", top: "150px", right: "5px" }}
          aria-label="Return to home page"
        />
      </Link>
```